bin/generate.js

```diff
@@ -0,0 +1,31 @@
+const Handlebars = require('handlebars');
+const fs = require('fs');
+const events = require(__dirname + '/../events.json');
+const template = fs.readFileSync(__dirname + '/../template.handlebars',  "utf-8");
+
+function render (source, events, url) {
+  let template = Handlebars.compile(source);
+  let output = template(events);
```

Comment:
I think inlining this would be neater as `template` its not used anywhere else.

`const output =  (Handlebars.compile(source))(events);`